File: project-euler/problem 25/solution.dart
void main(List<String> args) {
  List<BigInt> fibonacciSequence = [BigInt.from(1), BigInt.from(1)];

  while (fibonacciSequence.last.toString().length != 1000) {
    fibonacciSequence.add(fibonacciSequence[fibonacciSequence.length - 1] +
        fibonacciSequence[fibonacciSequence.length - 2]);
  }

  print(fibonacciSequence.length);
}

File: project-euler/problem 25/solution.py
def main():
    fibonacci_sequence: list[int] = [1, 1]


    while ( len(str(fibonacci_sequence[-1])) != 1000 ):
        fibonacci_sequence.append(
            fibonacci_sequence[-1] + fibonacci_sequence[-2]
        )

    print(len(fibonacci_sequence))


main()
